course-mw/enrolment-actor/src/main/scala/org/sunbird/enrolments/CourseEnrolmentActor.scala

```diff
@@ -359,6 +359,8 @@ class CourseEnrolmentActor @Inject()(@Named("course-batch-notification-actor") c
                 put(JsonKey.STATUS, ProjectUtil.ProgressStatus.NOT_STARTED.getValue.asInstanceOf[AnyRef])
                 put(JsonKey.DATE_TIME, new Timestamp(new Date().getTime))
                 put(JsonKey.COURSE_PROGRESS, 0.asInstanceOf[AnyRef])
+            } else {
+                logger.info(requestContext,"user-enrollment-null-tag, userId : "+userId+" courseId : "+courseId+" batchId : "+batchId + enrolmentData.toString);
```

Comment:
More changes are done to this object after this... 
to add logs - you should put the logger inside upsertEnrollment method